package.json

```diff
@@ -68,56 +68,56 @@
 		}
 	},
 	"dependencies": {
-		"@babel/core": "^7.12.10",
-		"@babel/plugin-proposal-class-properties": "7.12.1",
+		"@babel/core": "^7.24.6",
 		"@babel/plugin-syntax-import-meta": "^7.10.4",
-		"@babel/plugin-syntax-jsx": "^7.12.1",
-		"@babel/plugin-transform-flow-strip-types": "^7.12.10",
-		"@babel/plugin-transform-react-jsx": "^7.12.11",
-		"@babel/plugin-transform-regenerator": "^7.12.1",
-		"@babel/preset-env": "^7.12.11",
-		"@babel/preset-flow": "^7.12.1",
-		"@babel/preset-react": "^7.12.10",
-		"@rollup/plugin-alias": "^3.1.1",
-		"@rollup/plugin-babel": "^5.2.2",
-		"@rollup/plugin-commonjs": "^17.0.0",
+		"@babel/plugin-syntax-jsx": "^7.24.6",
+		"@babel/plugin-transform-class-properties": "7.24.6",
+		"@babel/plugin-transform-flow-strip-types": "^7.24.6",
+		"@babel/plugin-transform-react-jsx": "^7.24.6",
+		"@babel/plugin-transform-regenerator": "7.17.9",
+		"@babel/preset-env": "^7.24.6",
+		"@babel/preset-flow": "^7.24.6",
+		"@babel/preset-react": "^7.24.6",
+		"@rollup/plugin-alias": "^3.1.9",
+		"@rollup/plugin-babel": "^5.3.1",
+		"@rollup/plugin-commonjs": "^21.1.0",
 		"@rollup/plugin-json": "^4.1.0",
-		"@rollup/plugin-node-resolve": "^11.0.1",
-		"@surma/rollup-plugin-off-main-thread": "^2.2.2",
+		"@rollup/plugin-node-resolve": "^13.2.1",
+		"@surma/rollup-plugin-off-main-thread": "^2.2.3",
 		"asyncro": "^3.0.0",
-		"autoprefixer": "^10.1.0",
-		"babel-plugin-macros": "^3.0.1",
+		"autoprefixer": "^10.4.19",
+		"babel-plugin-macros": "^3.1.0",
 		"babel-plugin-transform-async-to-promises": "^0.8.18",
 		"babel-plugin-transform-replace-expressions": "^0.2.0",
 		"brotli-size": "^4.0.0",
-		"builtin-modules": "^3.1.0",
-		"camelcase": "^6.2.0",
+		"builtin-modules": "^3.2.0",
+		"camelcase": "^6.3.0",
 		"escape-string-regexp": "^4.0.0",
-		"filesize": "^6.1.0",
+		"filesize": "^8.0.7",
 		"gzip-size": "^6.0.0",
-		"kleur": "^4.1.3",
+		"kleur": "^4.1.5",
 		"lodash.merge": "^4.6.2",
 		"postcss": "^8.2.1",
-		"pretty-bytes": "^5.4.1",
-		"rollup": "^2.35.1",
+		"pretty-bytes": "^5.6.0",
+		"rollup": "^2.70.2",
 		"rollup-plugin-bundle-size": "^1.0.3",
-		"rollup-plugin-postcss": "^4.0.0",
+		"rollup-plugin-postcss": "^4.0.2",
 		"rollup-plugin-terser": "^7.0.2",
-		"rollup-plugin-typescript2": "^0.32.0",
-		"rollup-plugin-visualizer": "^5.6.0",
-		"sade": "^1.7.4",
-		"terser": "^5.7.0",
-		"tiny-glob": "^0.2.8",
+		"rollup-plugin-typescript2": "^0.36.0",
+		"rollup-plugin-visualizer": "^5.12.0",
+		"sade": "^1.8.1",
+		"terser": "^5.12.1",
+		"tiny-glob": "^0.2.9",
 		"tslib": "^2.0.3",
 		"typescript": "^4.1.3"
 	},
 	"devDependencies": {
-		"@babel/cli": "^7.12.10",
-		"@babel/node": "^7.12.10",
-		"@babel/plugin-proposal-throw-expressions": "^7.12.1",
+		"@babel/cli": "^7.24.6",
+		"@babel/node": "7.12.10",
```

Comment:
TODO: Bumping this requires altering our build command to ensure the flags are passed to Microbundle, not `babel-node`